api/src/main/java/io/strimzi/api/kafka/model/status/KafkaStatus.java

```diff
@@ -89,4 +90,18 @@ public String getKafkaMetadataVersion() {
     public void setKafkaMetadataVersion(String kafkaMetadataVersion) {
         this.kafkaMetadataVersion = kafkaMetadataVersion;
     }
+
+    @Description("Defines where cluster metadata are stored. Possible values are: " +
+            "ZooKeeper if the metadata are stored in ZooKeeper " +
+            "KRaftMigration if the controllers are connected to ZooKeeper, together with brokers, and the migration process is running " +
+            "KRaftDualWriting if the migration process finished and the cluster is in dual-write mode " +
+            "KRaftPostMigration if the brokers are fully KRaft-based but controllers being rolled to disconnect from ZooKeeper " +
+            "KRaft if the metadata are stored in KRaft")
```

Comment:
I think this is missing some separators:

```suggestion
            "ZooKeeper if the metadata are stored in ZooKeeper; " +
            "KRaftMigration if the controllers are connected to ZooKeeper, together with brokers, and the migration process is running; " +
            "KRaftDualWriting if the migration process finished and the cluster is in dual-write mode; " +
            "KRaftPostMigration if the brokers are fully KRaft-based but controllers being rolled to disconnect from ZooKeeper; " +
            "KRaft if the metadata are stored in KRaft.")
```